orders/models.py

```diff
@@ -1,3 +1,31 @@
+'''These are the order models'''
 from django.db import models
+from django.contrib import get_user_model
+from final_project import CustomModel
+
+AuthUserModel = get_user_model
 
 # Create your models here.
+'''A many to many relation with user to be defined here to be able to see the user through the order table'''
+class Order(CustomModel):
+    billing_address_id = models.ForeignKey(Address, on_delete=models.CASCADE)
+    user= models.ManyToManyField(User, through ='UserOrder', related_name='user')
```

Comment:
The relationship between `users` and `orders` shouldn't be many to many. Each order has it's own user.

Reply:
yes true. thank you!